repos/system_upgrade/el7toel8/actors/checkmemcached/actor.py

```diff
@@ -0,0 +1,62 @@
+from leapp.actors import Actor
+from leapp.models import InstalledRedHatSignedRPM
+from leapp.reporting import Report
+from leapp.tags import ChecksPhaseTag, IPUWorkflowTag
+from leapp.libraries.common.reporting import report_generic
+from leapp.libraries.stdlib import CalledProcessError, run
+
+import re
+
+class CheckMemcached(Actor):
+    name = 'check_memcached'
+    description = 'Check for incompatible changes in memcached configuration.'
```

Comment:
Can you please move actor description to the doc string? (example https://github.com/oamg/leapp-repository/blob/master/repos/system_upgrade/el7toel8/actors/checkxfs/actor.py)

Reply:
Updated. Thanks.